config/initializers/fog.rb

```diff
@@ -0,0 +1,15 @@
+CarrierWave.configure do |config|
+  if Rails.env.development? || Rails.env.test?
+    config.storage = :file
+  else
+    config.storage = :fog
+    config.fog_credentials = {
+      :provider               => 'AWS' || '',
+      :aws_access_key_id      => ENV['AWS_ACCESS_KEY_ID']|| '',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '||'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.